packages/dev/serializers/src/stl/stlSerializer.ts

```diff
@@ -25,7 +26,8 @@ export class STLExport {
         binary: boolean = false,
         isLittleEndian: boolean = true,
         doNotBakeTransform: boolean = false,
-        supportInstancedMeshes: boolean = false
+        supportInstancedMeshes: boolean = false,
+        mergeMeshes: boolean = true

```

Comment:
To keep backwards compatibility, you should use the opposite instead, something like `dontMergeMeshes = false`. This is not the best because there are two negations, so if you can think of a better name, feel free!

Reply:
Yes, because of this I used `mergeMeshes ` :( What about `preserveMeshes` or `individualMeshes` I have no idea?

Reply:
Great, thanks. I updated it